README.md

```diff
@@ -48,17 +48,18 @@ Step 1: Clone the repository
 Step 2: Download the datasets and move the data files to the dataset directory
 
   ```sh
-  wget -r -np -R "index.html*"  http://43.138.115.238/download/febench/data/; cp -r <dataset directory> ./dataset
+  wget -r -np -R "index.html*"  http://43.138.115.238/download/data/ -P ./dataset
```

Comment:
43.138.115.238 is valid? it should be https://www.openmldb.com/download/febench/data/? And https needs --no-check-certificate.
And if you download it, the tree is 
```
dataset
`-- www.openmldb.com
    |-- download
    |   `-- febench
    |       `-- data
    |           |-- Q0
```
You should download it to check.